assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -538,6 +538,9 @@ che.infra.openshift.trusted_ca.dest_configmap_labels=config.openshift.io/inject-
 # Additional labels to add into every Route created by Che server
 # to allow clear identification.
 che.infra.openshift.route.labels=NULL
+# The hostname that should be used as a suffix for the workspace routes.
+# For example host=open.che.org then the route will look like routed3qrtkptr0.open.che.org
+che.infra.openshift.route.host=NULL
```

Comment:
`host` implies the whole host including domain.
In this case it is domain.
`che.infra.openshift.route.domain' Let's be consistent with operator :)

Reply:
I would love to. However, in route spec it is `host` https://docs.openshift.com/container-platform/4.6/rest_api/network_apis/route-route-openshift-io-v1.html. Would you like to change it to the host in the operator?

Reply:
lets think about `che.infra.openshift.route.domain` or `che.infra.openshift.route.host.domain_suffix`

Reply:
`che.infra.openshift.route.host.domain_suffix` sounds ok for me